ush/templates/FV3.input.yml

```diff
@@ -96,6 +96,25 @@ FV3_HRRR:
     ldiag_ugwp: false
     lgfdlmprad: false
     thsfc_loc: false 
+    sfclay_compute_flux: true
+    # Smoke/dust options
+    rrfs_sd : false
```

Comment:
Will "rrfs_sd" be dynamically determined by DO_SMOKE_DUST?

Reply:
Currently it is not. We have to change both rrfs_sd and DO_SMOKE_DUST true or false. Hope we can make "rrfs_sd" dynamically determined by DO_SMOKE_DUST in next PR.